index.html

```diff
@@ -4,21 +4,25 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
     <title>Andrew's portfolio</title>
 </head>
 <body>
   <div class="container">
-      <div class="box1">MY Logo</div>
+      <div class="logo">
```

Comment:
- [ ]  It's required that we enclose this Logo in an anchor `<a>` tag. Please consider refactoring it. If you notice most websites have a logo so you can go back home page by clicking it 😃, that kind of UX convention make the navigation in the website way more pleasant and intuitive.